testfiles/1ZoneUncontrolled_win_2.idf

```diff
@@ -411,18 +411,6 @@
 
   Output:Variable,ZN001:WALL001:WIN001,Surface Outside Face Temperature,timestep;
 
-! Multiplier 2 causes a new window definition.
-
-  Output:Variable,ZN001:WALL001:WIN001:2,Surface Outside Face Incident Solar Radiation Rate per Area,timestep;
```

Comment:
Just for the record. Window multipliers do not create new windows, they just adjust the areas. Delete these output variable objects that trip warnings .
`** Warning ** The following Report Variables were requested but not generated -- check.rdd file`